package.json

```diff
@@ -14,6 +14,7 @@
     "phantomjs": "^1.9.16",
     "saucelabs": "^0.1.1",
     "selenium-standalone": "^4.4.0",
+    "guacamole": "^1.0.9",
```

Comment:
We're going to want to figure out where between `dependencies` and `devDependencies` this should be... And if `devDependencies` we'll need to add an installation note on the common install path because the `guacamole` option defaults to true.

For now, I've also switched us to a lazy `var guacamole = require("guacamole");` only if guacamole is enabled...